src/dataflow/decode/mod.rs

```diff
@@ -133,14 +135,17 @@ where
             stream,
             &enc.value_schema,
             enc.schema_registry_url,
-            envelope == Envelope::Debezium,
+            envelope.get_avro_envelope_type() == interchange::avro::EnvelopeType::Debezium,
```

Comment:
Does this have the same behavior as before?

Reply:
yes.

Also, this is an artifact of getting the split PR to compile and maintain whatever behaviors used to exist. You should probably just ignore this file entirely and pay attention to the version in #2493.